src/overlays/actors/ovl_Dm_Hina/z_dm_hina.h

```diff
@@ -10,10 +10,24 @@ typedef void (*DmHinaActionFunc)(struct DmHina*, PlayState*);
 typedef struct DmHina {
     /* 0x000 */ Actor actor;
     /* 0x144 */ DmHinaActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0x10];
+    /* 0x148 */ f32 unk148;
+    /* 0x14C */ f32 unk14C;
+    /* 0x150 */ f32 unk150;
+    /* 0x154 */ f32 unk154;
     /* 0x158 */ f32 unk_158;
     /* 0x15C */ f32 unk_15C;
```

Comment:
```suggestion
    /* 0x158 */ f32 unk158;
    /* 0x15C */ f32 unk15C;
```
To keep with the style of the other ones.